sw/device/tests/flash_ctrl_idle_low_power_test.c

```diff
@@ -130,9 +155,18 @@ bool test_main(void) {
         &pwrmgr, kDifPwrmgrWakeupRequestSourceTwo, 0));
 
     CHECK_DIF_OK(dif_aon_timer_watchdog_stop(&aon));
+
+    uint32_t bark_th = kAONBarkTh;
+    uint32_t bite_th = kAONBiteTh;
+
+    // Update bark and bite threshold in case of silicon test
+    if (kBootStage == kBootStageOwner) {
```

Comment:
Sounds like this should be a check against `kDeviceType`, not `kBootStage`, right?

Reply:
Replaced all (kBootStage, kBootStageOwner) with (kDeviceType , kDeviceSilicon).